stdlib/ssl.pyi

```diff
@@ -102,7 +102,15 @@ def RAND_egd(path: str) -> None: ...
 def RAND_add(__s: bytes, __entropy: float) -> None: ...
 def match_hostname(cert: _PeerCertRetType, hostname: str) -> None: ...
 def cert_time_to_seconds(cert_time: str) -> int: ...
-def get_server_certificate(addr: tuple[str, int], ssl_version: int = ..., ca_certs: str | None = ...) -> str: ...
+
+if sys.version_info >= (3, 10):
+    def get_server_certificate(
+        addr: tuple[str, int], ssl_version: int = ..., ca_certs: str | None = ..., timeout: float = ...
```

Comment:
This will keep failing stubtest, because `timeout` has type `object`. Can you add a comment to `stubtest_allowlists/py310.txt`?

Reply:
This is fine, for the same reason as in #6415